pkg/project/project.go

```diff
@@ -1,13 +1,18 @@
 package project
 
 var (
-	description        = "The aws-operator handles Kubernetes clusters running on a Kubernetes cluster inside of AWS."
-	gitSHA             = "n/a"
-	name        string = "aws-operator"
-	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "n/a"
+	bundleVersion        = "5.5.0"
```

Comment:
So after this one gets tagged, the next commit should increase this number, right?

Reply:
Yes for now. We are working towards removing this const.

Reply:
https://github.com/giantswarm/giantswarm/blob/master/processes/release-an-operator.md#release-a-new-operator-version-to-all-installations